openlibrary/core/models.py

```diff
@@ -275,17 +275,25 @@ def is_in_private_collection(self):
         """
         return private_collection_in(self.get_ia_collections())
 
-    def can_borrow(self):
+    def in_borrowable_collection(self):
         collections = self.get_ia_collections()
         return ('lendinglibrary' in collections or
             ('inlibrary' in collections and inlibrary.get_library() is not None)
             ) and not self.is_in_private_collection()
 
+    def can_borrow(self):
+        """This method should be deprecated in favor of in_borrowable_collection"""
+        return self.in_borrowable_collection()
+
     def get_waitinglist(self):
         """Returns list of records for all users currently waiting for this book."""
         return waitinglist.get_waitinglist_for_book(self.key)
 
-    def get_waitinglist_size(self):
+    def get_ia_waitlist_size(self):
+        availability = lending.get_availability_of_ocaid(self.get('ocaid'))
+        return int(availability.get('responses', {}).get(self.get('ocaid'), {}).get('num_waitlist', 0))
```

Comment:
this return looks like its doing a lot of work, maybe `lending` should handle the logic for extracting the waitlist size?